src/Schemes/AffineSchemes/Morphisms/Methods.jl

```diff
@@ -0,0 +1,148 @@
+export restrict, fiber_product, product, simplify
+
+
+
+###########################################################
+# (1) The fibre product of two morphisms of affine schemes
+###########################################################
+
+@Markdown.doc """
+    fiber_product(f::SpecMor{SpecType, SpecType, <:Any}, g::SpecMor{SpecType, SpecType, <:Any}) where {SpecType<:StdSpec}
+
+For morphisms ``f : Y → X`` and ``g : Z → X`` return the fiber
+product ``Y ×ₓ Z`` over ``X`` together with its two canonical
```

Comment:
```suggestion
product  $Y \times_X Z$ over ``X`` together with its two canonical
```
The unicode looks too wierd to be an improvement.

Reply:
I understand that @HechtiDerLachs uses a keyboard layout that allows him to directly enter these symbols. Some of them (e.g. the element symbol or the mapping arrow symbols above) I found rather nice. But I agree that this here is somewhat small and it is probably easier to read the LaTeX-code.